trpc/router/whatsnew.ts

```diff
@@ -18,11 +20,13 @@ export const whatsnew = router({
             let read = false;
             let createdAt: Date | undefined;
 
-            const rawReleaseVersion = await fetch(`${process.env.PUBLIC_URL}/version.txt`).then((res) => res.text());
-            const version = rawReleaseVersion.replace(/\n/g, '');
+            const whatsnewDir = path.join(process.cwd(), 'src/pages/whatsnew');
+            const versionsWithNotes = fs.readdirSync(whatsnewDir).filter((f) => /^\d+\.\d+\.\d+$/.test(f));
```

Comment:
Looks non-safety
First of all this path could be available on an another path
Also function call may be finished with error

Reply:
Added try/catch